setup.py

```diff
@@ -221,10 +201,10 @@
                 extras_require[new_key] = extras_require.pop(key)
 
 everything = set()
-for key, deps in extras_require.items():
+for key, deps in sorted(extras_require.items()):
     if ':' not in key:
         everything.update(deps)
-extras_require['all'] = everything
+extras_require['all'] = list(sorted(everything))
```

Comment:
sorted already returns a list!

Reply:
Ih, I'm not sure hy I shoudl it returned a genrator.

Reply:
Damn I can't type today.